tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.TagsListGenerator.TagListGenerator/HotChocolateTags.g.cs

```diff
@@ -0,0 +1,124 @@
+﻿// <auto-generated/>
+#nullable enable
+
+using Datadog.Trace.Processors;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
+{
+    partial class HotChocolateTags
+    {
+        // SpanKindBytes = System.Text.Encoding.UTF8.GetBytes("span.kind");
+        private static readonly byte[] SpanKindBytes = new byte[] { 115, 112, 97, 110, 46, 107, 105, 110, 100 };
+        // InstrumentationNameBytes = System.Text.Encoding.UTF8.GetBytes("component");
+        private static readonly byte[] InstrumentationNameBytes = new byte[] { 99, 111, 109, 112, 111, 110, 101, 110, 116 };
```

Comment:
This is the duplication I was talking about earlier, if we replace `HotChocolateTags`, `GraphQLTags` and `GraphQLTagsBase` with a single type like below, it removes a lot of this duplication

```csharp
internal partial class GraphQLTags : InstrumentationTags
{
    public GraphQLTags(string instrumentationName)
        => InstrumentationName = instrumentationName;

    [Tag(Trace.Tags.InstrumentationName)]
    public abstract string InstrumentationName { get; }
```